### ISSUE TITLE
GPL-596 Flag/remove destroyed plates early in CP process

### ISSUE THREAD

**Issue by rl15** [sanger/crawler#91]
**User story**
GPL-596 | As SSR (Debbie) I would like stock plates that have been destroyed to be be flagged / removed automatically from the cherry picking process (not just LH but useful for LH) [RT 695387]

**Who are the primary contacts for this story**
Debbie P
Danni W

**Acceptance criteria**

Token for a conversation to have. I think knowledge of a plate destruction is held in LabWhere, inferred from a locations naming. eg"Ogilivie - Destroyed"

Q. Should destruction be a attribute of the plate?
Q. If a labware is destroyed in LabWhere could the LIMS also be updated?

To be considered successful the solution must allow:
- [ ] add a list of acceptance criteria here
- [ ] ...

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**
Debbie P wrote (Wed, 29 Jul 2020 09:08:21 +0000) [RT 695387]

I was generating cherry picks for faculty work yesterday and came across this – 2 stock plates which we no longer have on site were included in the cherry pick (see pic below)

It would be good if these could be flagged earlier on in the cherry pick process so that they are not included.

This could also have value for Lighthouse cherry picks if we get to the stage of destroying plates once cherry picked, so that they are not included in future picks.

**Comment by rl15** [comment]
![10286412](https://user-images.githubusercontent.com/7222395/89173033-85c20480-d57b-11ea-981c-cbee6ce9ed01.png)

**Comment by rl15** [comment]
Agreed low priority in refinement session 4th Aug

**Comment by rl15** [comment]
Discussed with Rich C & James M on 20th October agreed to deprecate/archive low value given lens of Beckman robotics
